fixtures/builder.py

```diff
@@ -40,6 +41,13 @@ def __init__(self, name):
 
         self.repository.status()
 
+    def __del__(self):
+        # Create a hash for the generated fixture.
+        file_contents = "hash:" + dirhash(self.dir, 'md5')
+        hash_file = open(self.dir + "/hash.txt", "w")
```

Comment:
We should use Python's `with open() as hash_file` construct here; if I remember correctly, it does automatic clean-up.

Reply:
I didn't know about `with open() as`
fixed